src/windows/bindings/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    windows::build! {
+        Windows::Win32::NetworkManagement::IpHelper::{GetIpAddrTable,GetInterfaceInfo,GetAdaptersAddresses},
+        Windows::Win32::Networking::WinSock::{SOCKADDR_IN,SOCKADDR_IN6},
+        Windows::Win32::System::Diagnostics::Debug::*
+        //Windows::Win32::System::Diagnostics::Debug::{ERROR_BUFFER_OVERFLOW, ERROR_INSUFFICIENT_BUFFER, NO_ERROR}
+    };
+}
```

Comment:
Can we have some doc comments on this?

Reply:
The bindings sub crate is somewhat irrelevant.
It's there only to improve compilation time, winwows-rs recomends this to act as a form of cache.
You only specify the windows apis to generate for your project.